src/main/java/frc/robot/subsystems/shooter/SetShooterSpeed.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Robot;
+import java.util.function.DoubleSupplier;
+
+import static frc.robot.Robot.shooter;
+
+
+/**
+ * This command spins the wheel in the desired velocity in order to shoot the power cells.
+ */
+public class SetShooterSpeed extends CommandBase {
+
+    private DoubleSupplier velocitySetpoint;
+    private double lastSetpoint;
+
+    /**
+     * Constructs a shoot command with default RPM setpoint.
+     */
```

Comment:
* Constructs a shoot command with default RPM setpoint.
add @see the default RPM in robot constans